src/components/styles/NoxThemeGen.js

```
@@ -0,0 +1,29 @@
+import {
+  argbFromHex,
+  themeFromSourceColor,
+  applyTheme,
+  hexFromArgb,
+} from '@material/material-color-utilities';
+
+const convertColor = color => {
+  if (typeof color === 'object') {
+    const newColor = {};
+    Object.keys(color).forEach(k => (newColor[k] = convertColor(color[k])));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)